tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27-test, py37-test, py38-test, py27-flake8, docs
+envlist = py27-test, py37-test, py27-flake8, docs
```

Comment:
Not sure why this is showing up in the diff? `py38-test` is disabled by default on master.